### ISSUE TITLE
Ability to copy bundle product types into other targets

### ISSUE THREAD

**Issue by keith** [yonaskolb/XcodeGen#91]
With this configuration:

```yaml
name: Project
targets:
  Project:
    type: framework
    platform: iOS
    dependencies:
      - target: ProjectResources
  ProjectResources:
    type: bundle
    platform: iOS
```

You can create a framework target, and an associated resource bundle for that target. But it doesn't seem like there's any way for you to add `ProjectResources` in this case to the copy resources phase of `Project`.

This would be very nice to support for the case where you have static libraries and resource bundles, since that's the only way to vend resources and not have conflicts with the targets linking the final library.

**Comment by keith** [comment]
I've submitted a possible fix to this here: https://github.com/yonaskolb/XcodeGen/pull/95

**Comment by jerrymarino** [comment]
@yonaskolb @keith I think I've got a similar issue. I'm trying to add some pre built bundles from various Vendors to my iOS App. It looks like merely including a bundle in a targets `sources` won't work. I'm not sure if pre built bundles should be treated as `sources`, but it is aligned with how other resources are implemented

![prebuiltbundles](https://user-images.githubusercontent.com/1245820/32355291-0d8f0de4-bfeb-11e7-9875-e30a8bad47f5.png)

**Comment by yonaskolb** [comment]
@jerrymarino, your issue should be resolved in #148
